content/hardware/04.pro/boards/portenta-x8/compatibility.yml

```diff
@@ -2,8 +2,6 @@ software:
   - arduino-ide
   - arduino-cli
 hardware:
-  shields:
-    - portenta-vision-shield
   boards:
     - portenta-breakout
```

Comment:
Thanks @jhansson-ard! Can we also move Portenta Breakout under Carriers section? Then it's perfect!